### DISCUSSION TITLE
目前项目使用 nvm 为主，装了自动识别node 版本的插件，如何无缝衔接到 nvm-desktop

### DISCUSSION THREAD

**Discussion by xmasuhai** [1111mp/nvm-desktop#43] [Q&A]
如何无痛迁移到 nvm-desktop，或者写脚本，类似自动识别项目中的 .nvmrc，切换 node 版本

**Comment by 1111mp** [comment]
我不确定`nvm`安装`node`的目录结构，但是`nvm-dsktop`安装node的话跟你自己去官网下载到本地的方式是一样的，只不过这一切都被`nvm-desktop`托管了。

目前可能没有办法无痛迁移过去（实现原理完全不一样，而且`nvm-desktop`是通过项目目录下的`.nvmdrc`文件识别正确的`node`版本）,这一点十分抱歉。但是两者可以同时使用，不会相互冲突，只不过需要改变其对应在系统环境变量中的优先级（哪个优先级高，系统会识别到就会使用哪个）。
如果现有项目使用的是`nvm`的话，你自己本地需要使用`nvm-desktop`，那么你可以为项目指定对应的node版本（`.nvmdrc`），然后将其添加到`.gitignore`文件，这样就不会影响其他人的合作。
